### PR TITLE
footer.php: Add the Docker Hub tags filter link to "Docker Tag"

### PR DESCRIPTION
**Pull request by PeterDaveHello**
**By submitting this pull request, I confirm the following:** 

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/AdminLTE/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [ ] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
- [x] I have Signed Off all commits. ([`git commit --signoff`](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff))

---

**What does this PR aim to accomplish?:**

There're some links to the releases pages at the footer, which is very useful, but only the Docker Tag doesn't have it:
image

**How does this PR accomplish the above?:**

Just add the Docker Hub link to filter the Docker image tags in the `pihole/pihole` repository, result:

**Docker Tag** [2022.01.1](https://hub.docker.com/r/pihole/pihole/tags?page=1&name=2022.01.1)
image

**What documentation changes (if any) are needed to support this PR?:**

Doesn't look like it need documentation ;)

### CODE REVIEW

File: scripts/pi-hole/php/footer.php [hunk 1]
```diff
@@ -59,6 +59,11 @@
     $coreReleasesUrl = $coreUrl . "/releases";
     $ftlReleasesUrl = $ftlUrl . "/releases";
     $webReleasesUrl = $webUrl . "/releases";
+
+    $dockerhubBaseUrl = "https://hub.docker.com/r/pihole";
+    $dockerRepoUrl = $dockerhubBaseUrl . "/pihole";
+    $dockerTagsUrl = $dockerRepoUrl . "/tags";
+    $dockerTagsFilterUrl = $dockerTagsUrl . "?page=1&name=";
```
Comment (rdwebdesign) [inline scripts/pi-hole/php/footer.php]: **Note:**
The `page=1` is not necessary, since the first page is the default page.

**Comment by yubiuser** [comment]
I like the idea of adding a link to the docker tag. Why don't you link to the github page as for all the other components?

https://github.com/pi-hole/docker-pi-hole/releases

File: scripts/pi-hole/php/footer.php (hunk 1 above)
Comment (PeterDaveHello) [reply scripts/pi-hole/php/footer.php]: Yeah! I know that! Just we don't have control on Docker Hub, so I don't know the spec and defined default behavior would be changed or not, simply following the behavior we found on the webpage to have the parameter seems safer to me!

**Comment by PeterDaveHello** [comment]
@yubiuser though there's a release page for Docker images on GitHub, the page really reflects the published images is on the Docker Hub, so I chose Docker Hub, what do you think?

**Comment by yubiuser** [comment]
Looking at the link you posted above I prefer the link to the github page as it contains release notes.

**Comment by PromoFaux** [comment]
I'm inclined to agree that it should probably point at the github releases page - however, for `:dev` and `:nightly` there are no release notes there (to be expected - they are rolling images)

**Comment by rdwebdesign** [comment]
> Just we don't have control on Docker Hub, ...

That's one more reason to point to the github page.

**Comment by PeterDaveHello** [comment]
How do we deal with the :dev and :nightly tag then?

**Comment by yubiuser** [comment]
They don't get a link. It's the same with non-docker installations running custom branches

Bildschirmfoto zu 2022-01-22 07-23-25

**Comment by PeterDaveHello** [comment]
Cool, let me revise the patch.

**Comment by PeterDaveHello** [comment]
Should be good now.

**Review by PromoFaux — APPROVED** [review]
Tested on a fresh container (with some modifications to pull in your branch) and confirmed working:

works

**Pull request merged**